File: Segundo-Bimestre/Parcial 1/Arreglos Unidimensionales/ArreglosUnidimensionales_java/src/arreglosunidimensionalesfc/NumEnteroVector.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package arreglosunidimensionalesfc;
import java.util.Scanner;
/**
 *
 * @author DELL
 */
public class NumEnteroVector {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        
        // Declarar e inicializar las variables
        int i, n, num, cont;
        n = 0; num = 0; cont = 0;
        
        System.out.print("Ingrese el tamaño del vector: ");
        n = leer.nextInt();
        
        System.out.print("Ingrese un número entero: ");
        num = leer.nextInt();
        
        // Declarar el vector
        int [] A = new int [n];
        
        for (i=0;i<n;i++) {
            System.out.print("Ingrese el elemento A["+i+"]: ");
            A[i] = leer.nextInt();
            if (A[i] == num) {
                cont = cont + 1;                
            }
        }
        
        for (i=0;i<n;i++) {
            if (A[i] == num) {
                System.out.println("El número entero "+num+" se ha repetido en la posición: A["+i+"]");
            }
        }        
        System.out.println("El número entero " +num+ " se ha repetido "+cont+" veces.");        
    }
}

File: Segundo-Bimestre/Parcial 1/Arreglos Unidimensionales/ArreglosUnidimensionales_python/NumEnteroVector.py
print("")
# Inicializar las variables
i = 0; n = 0; num = 0; cont = 0

n = int(input("Ingrese el tamaño del vector: "))
num = int(input("Ingrese un número entero: "))

# Declaramos los vectores
A = [None]*n

for i in range (0, n):
    A[i] = int(input("Ingrese el elemento A["+str(i)+"]: "))
    if (A[i] == num):
        cont = cont + 1

for i in range (0, n):
    if (A[i] == num):
        print("El número entero ",num," se ha repetido en la posición: A["+str(i)+"]")

print("El número entero ",num," se ha repetido ",cont," veces")